### PR TITLE
To Do List (Phase 3)

### PR DESCRIPTION
**Pull request by katarighe**
In this final phase, the following tasks are done:

1. **Added** a new JavaScript file and imported as a module, which is `taskStatus.js`. this file contains the methods related to the status updates of the To Do List, such as `(completed: true / false)`.

2. **Added** event listener to the checkbox, which now displays as `check-icon` and `uncheck-icon`.

3. **Updated** the items objects value for completed key upon the user interacts with it. 

4. **Implemented** a JS function for the _"Clear all completed"_ button, which is `removeCompletedTasks`.

5. **Stored** the updated array of items in the _local storage_, so it will allow the user to retrieve the correct list of values after reloading the page. This means that any made changes to the list is preserved as well. 

6. **Added** JS functions on the _"Add"_ icon. This function adds the task directly to the list when the user clicks it. 

7. **Added** JS functions on the _"Refresh"_ icon, as well as including these functions on the HTML tag. This function refreshes the current page when the user clicks. Any data made to the To Do List is preserved inside the local storage.

8. **Made** additional changes to the theme and the CSS, such as changing the colors of the background and hover, as well as aligning and adjusting text and the container.  

**Update [13:23 - UTC+1]**
- **Added** mouseover title feature on each To Do List elements, which provides a description when hovering an icon or a text. 

**Final outcome of Phase 3:** 
image

### CODE REVIEW

File: src/modules/functions.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-let myTask = JSON.parse(localStorage.getItem('myTask')) || [];
+let myTask = JSON.parse(localStorage.getItem('myTask')) || []; // eslint-disable-line
```
Comment (OLIPLICHE) [inline src/modules/functions.js]: - I would advise you not to disable the linter. The[ linter](https://github.com/microverseinc/linters-config) is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Can you solve it manually?

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @katarighe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- It has an event listener on the checkbox 👍
- It contains methods related to the status updates(true/false) 👍
- The updated array of items is stored in local storage 👍
- `Clear all completed`, is implemented 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/functions.js (hunk 1 above)
Comment (katarighe) [reply src/modules/functions.js]: Removed and fixed. ✅

**Review by TadesseDev — APPROVED** [review]
### Hi @katarighe 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**